Assets/VRTK/Scripts/Interactions/VRTK_ControllerEvents.cs

```diff
@@ -80,6 +82,9 @@ public enum ButtonAlias
             Start_Menu_Press
         }
 
+        [Tooltip("The controller who's events will be propagated.")]
```

Comment:
Like above this should probably read something like "The hand of the controller to propagate events for. If this is set to `None` the game object this script is attached to will be used.".